The vty version constraint is to lax, it seems that hleder-ui is not
compatible with the 6.0 vty version. Hence we add an upper bound for
the version constraint.

--- hledger-1.31.orig/hledger-ui/hledger-ui.cabal
+++ hledger-1.31/hledger-ui/hledger-ui.cabal
@@ -97,7 +97,7 @@
     , transformers
     , unix
     , vector
-    , vty >=5.15
+    , vty >=5.15 && <6.0
   default-language: Haskell2010
   if os(windows)
     buildable: False
